### ISSUE TITLE
cannot read property 'toString' of undefined

### ISSUE THREAD

**Issue by gianciccarelli** [swdotcom/swdc-vscode#109]
**Describe the bug**
```Code Time plugin doesn't load properly and the data of daily metrics it doesn't change. 
```
**ISSUE TYPE**
- Bug Report

**Steps to reproduce**
1. Initialize vscode.
2. Click on Code Time section/panel.
3. Wait to load.
4. See error.

**Expected behaviour**
```
See section MENU loaded and DAILY METRICS with changes.
```

**ACTUAL RESULTS**
```
Just like the screenshot below. All I can see is the daily metrics without changes and the contributors section, the section menu it doesn't show. The daily metrics doesn't show any changes and when I view my summary the last time updated is for a day that I doesn't use the plugin code time, I used yesterday (July 11) and it shows April 21
```

**Screenshots**
![Screenshot from 2020-07-12 15-12-05](https://user-images.githubusercontent.com/39072813/87253541-663f2c80-c452-11ea-8e78-466252ae217b.png)

**Desktop (please complete the following information):**
 - OS: elementary OS 5.1.6 Hera x86_64
 - Plugin version:  Code Time v2.2.28

**Additional context**
```
I tried in another laptop with the same OS and the problem persist.
```

**Comment by xavluiz** [comment]
I apologize for the late reply and thank you for the issue report. I'll look into this today and let you know what we find.

**Comment by xavluiz** [comment]
We've updated code time with a fix for this in v2.2.32. We apologize for any inconvenience. Regards.
